otel/manifest.json

```
@@ -45,7 +45,8 @@
       }
     },
     "dashboards": {
-      "OpenTelemetry Dashboard": "assets/dashboards/dashboard.json"
+      "OpenTelemetry Host Metrics Dashboard": "assets/dashboards/otel_host_metrics_dashboard.json",
+      "OpenTelemetry Collector Metrics Dashboard": "assets/dashboards/otel_collector_metrics_dashboard.json"
```

The validation error is coming from here. Due to implementation details, it's strongly preferred that keys in this `"dashboards"` mapping do not change or be removed. Since it's not a public facing name, what do you think of reusing it like so:

```suggestion
      "OpenTelemetry Dashboard": "assets/dashboards/otel_host_metrics_dashboard.json",
      "OpenTelemetry Collector Metrics Dashboard": "assets/dashboards/otel_collector_metrics_dashboard.json"
```

---

That's fine as long as it's not public facing!